recipe/0068-newspaper/index.md

```diff
@@ -1,43 +1,253 @@
 ---
-title: A newspaper
+title: A basic newspaper
 id: 68
 layout: recipe
 tags: [tbc]
 summary: "tbc"
 ---
 
-
 ## Use Case
 
-Why is this pattern is important?
+Digitised Newspapers are more complicated than some other types of content as the hierarchy of Titles, Volumes and Issues are important in making them accessible. The date of publication and providing access to OCR data are also important to the viewing experience. This recipe gives an example of a basic Newspaper with two issues and links to text generated from Optical Character Recognition (OCR) software. The aim is to give a good outline of common mapping solutions for Newspapers. 
 
 ## Implementation notes
 
-How does one implement the pattern?
+This recipe was put together by Nuno Freire from Europeana and was discussed by the IIIF Newspaper Community Group. The example is a cut down Newspaper with two issues containing two pages. The OCR is at page level rather than article level. The example is for a German Newspaper called the ‘Berliner Tageblatt’ and is hosted by Europeana. In the IIIF domain a Newspaper Title is considered to be a __IIIF Collection__ and points to issue __IIIF Manifests__ through the `items` element. An issue in this example is considered to be the unit or edition that was published on a particular date. For a more complete discussion on Newspaper hierarchies and how they map to IIIF see the IIIF Newspaper Guidance notes. 
 
-## Restrictions
+### Date of publication or navDate
+One of the important features of Newspapers is the publication date which can allow the user to navigate issues by the date they were published. To achieve this we use the special [navDate](https://iiif.io/api/presentation/3.0/#navdate) property. This `navDate` should be added both to the items in a collection and also in the issue manifest. This allows viewers to present a date based navigation for Newspaper collections. 
 
-When is this pattern is usable / not usable? Is it deprecated? If it uses multiple specifications, which versions are needed, etc.? (Not present if not needed.)
+The `navDate` is added to the collection in the items property for example:
 
-## Example
+Full JSON: [newspaper_title-collection.json](newspaper_title-collection.json)
+
+{: class="line-numbers line-highlight" data-line="10,20"}
+```json
+"items": [
+    {
+      "id": "https://../newspaper_issue_1-manifest.json",
```

Comment:
Examples should be directly usable. These `...` make it unusable. The pattern for other recipes is to include the full example in the document. I think the current example is thus an anti-pattern.

Reply:
To be discussed in the TRC: https://github.com/IIIF/trc/issues/33#issuecomment-564281810. There is a link to working JSON on line 22 and we need to decide going forward if this snippet structure is allowable going forward.